.github/workflows/ci.yaml

```diff
@@ -121,7 +121,7 @@ jobs:
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+          aws-region: 'eu-west-1'
```

Comment:
the aws region was taken from the pipeline as a github secret. There is no point doing it -- we are hardcoding the region for the other Cellenics repositories.